app/components/moderation-base/template.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="dark-header"> {{!HEADER}}
+    <div class="container"> {{!CONTAINER}}
+        <div class="row">{{!HEADER ROW}}
+            <div class="text-left p-l-lg col-md-12">
+                <p>
+                    {{#each breadCrumbs as |breadCrumb|}}
```

Comment:
Would it be better to use an existing bread crumbs library? Do you know if we already use one?